docs/node-support.md

```diff
@@ -18,6 +18,7 @@
 | 13.x             | 3.15.0 - 3.20.0              |
 | 14.x             | 3.19.0 - Current             |
 | 16.x             | 3.28.0 - Current             |
+| 18.x             | 3.28.0 - Current             |
```

Comment:
```suggestion
| 18.x             | 4.6.0 - Current              |
```
I think.

Also, we might want to add v18 to the `Current support` section below.
Will check and let you know.

Reply:
For the `Current support` section you could add:

```md
* v18: Released as stable version.
  * Supported by ember-cli/ember-cli#master until: 2025-04-30.
```

As v18 is not yet an LTS release, but we can update the wording later on.